index.html

```diff
@@ -4,18 +4,18 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
   <title>Awesome Books</title>
 </head>
 <body>
-  <main>
-    <section id="add-books">
-			<h2>Add a new Book</h2>
-			<div class="bookInput">
-				<input type="text" id="title" name="title" placeholder="Title">
-				<input type="text" id="author" name="author" placeholder="Author">
-				<button type="button" class="add-book-btn">Add</button>
-			</div>
-		</section>
-  </main>
+  <h1>Awesome books</h1>
+	<div class="books"></div>
+	<div>
+		<input type="text" id="title" name="title" placeholder="Title">
+		<input type="text" id="author" name="author" placeholder="Author">
+		<button type="button" class="add-book-btn">Add</button>
+	</div>
+  <script src="script.js"></script>
+
```

Comment:
- [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.